bears/js/JSHintBear.py

```diff
@@ -43,6 +43,9 @@ class JSHintBear:
 
     @staticmethod
```

Comment:
commit message: please enclose globalstrict in double backticks. i think coala should be too.